packages/symreg/src/symreg/genetic_programming.py

```diff
@@ -42,28 +43,35 @@ def setup_environment(self):
         pset.addPrimitive(operator.add, 2)
         pset.addPrimitive(operator.sub, 2)
         pset.addPrimitive(operator.mul, 2)
+        pset.addPrimitive(operator.neg, 1)
         pset.addPrimitive(self.protectedDiv, 2)
-        pset.addPrimitive(math.sin, 1)
-        pset.addPrimitive(math.cos, 1)
-        pset.addEphemeralConstant("rand101", functools.partial(random.uniform, -1, 1))
```

Comment:
I'm taking these operators out since they can add unnecessary complexity to the solution trees, especially when the training examples (`training_data.csv`) are based on a simple polynomial.